src/main/groovy/org/mockito/release/notes/internal/DefaultImprovement.java

```diff
@@ -87,4 +87,40 @@ public String toJson() {
     public void toJson(Writer writable) throws IOException {
         writable.append(toJson());
     }
+
+    @Override
+    public boolean equals(Object o) {
```

Comment:
Are the equals/hashcode methods meaningful to fix the issue? Ideally, we don't have to maintain those methods unless we have to. They add complexity.